repos/spack_repo/builtin/packages/kokkos_nvcc_wrapper/package.py

```diff
@@ -159,10 +159,6 @@ class KokkosNvccWrapper(Package):
         deprecated=True,
     )
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
```

Comment:
do we want to keep the depends on cxx?
we do have a depends on Cuda below

Reply:
you need the `depends_on("cxx", type="build")` for `self.compiler.cxx` to exist.

Reply:
I agree, I could test without `depends_on("cxx", type="build")` and it indeed fails. Adding it makes it work again.